netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -110,7 +110,7 @@
       boolean permitKeepAliveWithoutCalls, long permitKeepAliveTimeInNanos,
       InternalChannelz channelz) {
     this.address = address;
-    this.channelType = checkNotNull(channelType, "channelType");
+    this.channelType = channelType == null ? Utils.defaultServerChannel() : channelType;
```

Comment:
Why not initialize `channelType` to `defaultServerChannel()`?

Reply:
no longer relevant, but i think the channelType in the server is final.  default value for eventloop was handled here. which is moved to the builder in newer commit. PTAL